LeetCodeAssistant/LeetCodeProblemRepository.swift

```diff
@@ -205,6 +236,23 @@ class LeetCodeProblemRepository {
             var number: Int
             var difficulty: LeetCodeProblemDifficuly
             var title: String
+            var description: String
+        }
+    }
+    
+    private struct JsonData: Codable {
```

Comment:
JsonData is too unclear name and possible to conflict for the name with other structs. Please change the name to be proper one.